src/graphnet/training/utils.py

```diff
@@ -2,7 +2,7 @@
 
 from collections import OrderedDict
 import os
-from typing import Dict, List, Optional, Tuple, Union
+from typing import Dict, List, Optional, Tuple, Union, Any
```

Comment:
Change is not pertinent to the goal of the PR.
```suggestion
from typing import Dict, List, Optional, Tuple, Union
```